dev/how-to/migrate/from-mariadb-galera-cluster.md

```diff
@@ -0,0 +1,571 @@
+
+- NIC bonding or teaming
+- PCIe SSDs (not SATA SSDs)
+- CPU power saving management disabled in BIOS
+- NUMA disabled in BIOS (`flush_caches` is optimized in Percona Server 5.7.10. For details, see [Improved NUMA support](https://www.percona.com/doc/percona-server/5.7/performance/innodb_numa_support.html))
+
+### Operating system requirements
+
+- Use CentOS 6.9 64-bit or CentOS 7.3-1611 64-bit as the operating system.
+
+- If you use a virtual machine, do not deploy a cluster on a single physical machine.
+
+- Use XFS as the file system.
+
+- Disable SWAP space by setting `vm.swappiness` to 0.
+
+- If the memory size exceeds 64 GB, enable huge pages.
+
+- If the memory exceeds 32 GB, disable NUMA in the operating system.
+
+- Disable IPv6.
+
+    ```
+    # vim /etc/sysctl.d/99-sysctl.conf
+    net.ipv6.conf.all.disable_ipv6 = 1
+    net.ipv6.conf.default.disable_ipv6 = 1
+    ```
+
+- Enable port forwarding.
+
+    ```
+    # vim /etc/sysctl.d/99-sysctl.conf
+    net.ipv4.ip_forward = 1
+    ```
+
+- Modify `limits`.
+
+    ```
+    # vim /etc/security/limits.conf
+    mysql soft nofile 65535
+    mysql hard nofile 65535
+    ```
+
+- Modify `/etc/pam.d/login`.
+
+    ```
+    # vim /etc/pam.d/login 
+    session required /lib64/security/pam_limits.so
+    ```
+
+- Configure the MariaDB yum repository.
+
+    > **Note:**
+    >
+    > Currently, Data Migration supports only MariaDB 10.1.2 and later.
+
+    ```
+    Create mariadb.repo.
+    # vi /etc/yum.repo.d/mariadb.repo
+    Add the following (the official repository)
+    [mariadb-main]
+    name = MariaDB Server
+    baseurl = https://downloads.mariadb.com/MariaDB/mariadb-10.3/yum/rhel/$releasever/$basearch
+    gpgkey = file:///etc/pki/rpm-gpg/MariaDB-Server-GPG-KEY
+    gpgcheck = 1
+    enabled = 1
+
+    [mariadb-maxscale]
+    # To use the latest stable release of MaxScale, use "latest" as the version.
+    # To use the latest beta (or stable if there is no beta currently) release of MaxScale, use "beta" as the version.
+    name = MariaDB MaxScale
+    baseurl = https://downloads.mariadb.com/MaxScale/2.3/centos/$releasever/$basearch
+    gpgkey = file:///etc/pki/rpm-gpg/MariaDB-MaxScale-GPG-KEY
+    gpgcheck = 1
+    enabled = 1
+
+    [mariadb-tools]
+    name = MariaDB Tools
+    baseurl = https://downloads.mariadb.com/Tools/rhel/$releasever/$basearch
+    gpgkey = file:///etc/pki/rpm-gpg/MariaDB-Enterprise-GPG-KEY
+    gpgcheck = 1
+    enabled = 1
+    ```
+
+- Install the following software.
+
+    ```
+    # yum install iotop sysstat -y
+    ```
+
+## Install MariaDB
+
+You need to install the following on all the nodes in the database.
+
+1. Install MariaDB.
+
+    ```
+    # yum install MariaDB-server MariaDB-client galera mariabackup socat –y
+    ```
+
+2. Install `percona-toolkit`. It is a collection of advanced command-line tools, used for performance tuning, and it is an essential tool for DBAs.
+
+    ```
+    # yum install percona-toolkit –y
```

Comment:
Notice that there are 3 `yum install` commands in a row here. Why not combine them together into a single command so a user doesn't have to keep copying/pasting this stuff and waiting for yum to do its thing?